frontend/src/metabase/parameters/components/widgets/NumberInputWidget/NumberInputWidget.tsx

```diff
@@ -54,19 +58,22 @@ function NumberInputWidget({
 
   return (
     <WidgetRoot className={className}>
+      {title && <WidgetTitle>{title}</WidgetTitle>}
```

Comment:
Should it maybe be an HTML label? 🤔

Reply:
Good catch, should definitely be a label. Fixed.